js/renderingPicture.js

```diff
@@ -0,0 +1,37 @@
+import {createPhotos} from './mockPhotos.js';
+
+const dataCreatePicture = createPhotos();
+const SAMPLE = document.querySelector('#picture').content.cloneNode(true);
+const PICTURES = document.querySelector('.pictures');
```

Comment:
В camelCase. Типо такого, можно и просто `pictures` если тебе точно понятно что это такое
```suggestion
const picturesContainer = document.querySelector('.pictures');
```

Reply:
+